index.js

```diff
@@ -18,6 +18,7 @@ function gulpEslint(options) {
 
 	function verify(str, filePath) {
 		var result = linter.executeOnText(str, filePath).results[0];
+		result.filePath = path.relative(process.cwd(), result.filePath);
```

Comment:
Better fixed in https://github.com/adametry/gulp-eslint/commit/21cf2ef475c598aa82d56ba496e62f9764f547ce. Absolute paths are more useful for end users.